alf/environments/gym_wrappers.py

```diff
@@ -689,3 +689,48 @@ def __init__(self, env):
     def step(self, action):
         ob, reward, done, info = self.env.step(action)
         return ob, reward, False, info
+
+
+@alf.configurable
+class DiscreteActionWrapper(gym.ActionWrapper):
+    """Discretize each continuous action dim into several evenly distributed
+    values. Currently only support unnested action space as ``gym.spaces.Box``
+    with a rank-1 shape.
+    """
+
+    def __init__(self, env: gym.Env, actions_num: int):
```

Comment:
actions_num: the name could be more precise (e.g. bin_num), to reduce ambiguity with action dimensions.

Reply:
> actions_num: the name could be more precise (e.g. bin_num), to reduce ambiguity with action dimensions.

I think the the docstring is clear enough. It's not bin number as this has nothing to do with histogram.